internal/cli/events/list.go

```diff
@@ -65,25 +81,27 @@ func (opts *ListOpts) Run() error {
 	return opts.Print(r)
 }
 
-func (opts *ListOpts) newEventListOptions() *atlas.EventListOptions {
-	return &atlas.EventListOptions{
-		ListOptions: atlas.ListOptions{
-			PageNum:      opts.PageNum,
-			ItemsPerPage: opts.ItemsPerPage,
-		},
-		EventType: opts.eventType,
-		MinDate:   opts.minDate,
-		MaxDate:   opts.maxDate,
-	}
-}
-
-// mongocli atlas event(s) list [--projectId projectId] [--orgId orgId] [--page N] [--limit N] [--minDate minDate] [--maxDate maxDate].
+// ListBuilder
+// 	mongocli atlas event(s) list
+// [--projectId projectId]
+// [--orgId orgId]
+// [--page N]
+// [--limit N]
+// [--minDate minDate]
+// [--maxDate maxDate].
 func ListBuilder() *cobra.Command {
 	opts := &ListOpts{}
 	opts.Template = listTemplate
 	cmd := &cobra.Command{
-		Use:     "list",
-		Short:   "List events for an organization or project.",
+		Use:   "list",
+		Short: "Return all events for an organization or project.",
+		Deprecated: `  
+  To return project events prefer
+  $ mongocli atlas|ops-manager|cloud-manager events projects list [--projectId <projectId>]
+
+  To return organization events prefer
+  $ mongocli atlas|ops-manager|cloud-manager events organizations list [--orgId <orgId>]
+`,
```

Comment:
Question: I don't think this currently displays in the RST output. Do we want it to?

Reply:
I think the current behaviour is to ignore deprecated commands, that works for me if it works for docs

Reply:
Works for me.